apps/files/docs/extensionpoints.adoc

```diff
@@ -17,3 +17,29 @@
 
 OC.$extend.request('files', 'file-details-panel', payload).then( response => {});
 ----
+
+### Files list status indicators
+
+An extension can extend status indicators in files list with custom component, properties and methods.
+Extending this list can be done by creating a new object inside of appInfo called `filesListIndicators`. This object can contain keys `name` and `component`.
+
+#### Example:
+
+[source,js]
+----
+const appInfo = {
+  name: 'StatusIndicators',
+  id: 'status-indicators',
+  icon: 'info',
+  isFileEditor: false,
+  extensions: [],
+  filesListIndicators: [{
+    name: 'indicators',
+    component: Indicators
```

Comment:
any hint about what properties the "Indicators" vue is supposed to take as input ? what events can/should it trigger if any ?

Reply:
can you also rename it to "ExampleIndicators" to make it clearer which part of this code is referencing actual Phoenix code and what part is custom ?

Reply:
Renamed and added props into docs. Haven't added events though since there are none for custom indicators. If we want to introduce some we need requirements for them. At the moment, every method needs to be implemented and taken care of inside of the custom indicators components.